src/JustEat.StatsD/IStatsDTcpClient.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace JustEat.StatsD
+{
+    public interface IStatsDTcpClient : IStatsDClient
+    {
+    }
```

Comment:
The way I see it is that we would now have two working implementations of `IStatsDClient` - a name that says nothing about the transport. Another interface with no methods adds nothing.